packages/react-switch/src/components/Switch/useSwitchStyles.ts

```diff
@@ -2,9 +2,9 @@ import { makeStyles, mergeClasses } from '@fluentui/react-make-styles';
 import { createFocusOutlineStyle } from '@fluentui/react-tabster';
 import type { SwitchState } from './Switch.types';
 
-const rootClassName = 'ms-Switch-root';
-const trackClassName = 'ms-Switch-track';
-const thumbClassName = 'ms-Switch-thumb';
+export const switchClassName = 'fui-Switch';
+const trackClassName = `${switchClassName}-track`;
+const thumbClassName = `${switchClassName}-thumb`;
```

Comment:
Should we export these too?

Reply:
May be, but not in this PR. If we would like to do this, we should have some naming convention/consistency around it, see https://github.com/microsoft/fluentui/issues/19937#issuecomment-954581186.